docs/spec/callables.rst

```diff
@@ -347,60 +403,394 @@ generate errors in such cases.
+        f9: IntArgs = str_args  # Error: int is not subtype of str
+        f10: Standard = int_str_args  # Error: keyword parameters a and b missing
+        f11: Standard = str_args  # Error: keyword parameter b missing
+
+
+``**kwargs`` parameters
+^^^^^^^^^^^^^^^^^^^^^^^
+
+If a callable ``B`` has a signature with a ``**kwargs`` parameter (without
+an unpacked ``TypedDict`` type annotation), callable ``A`` must also have a
+``**kwargs`` parameter to be a subtype of ``B``, and the type of
+``B``'s ``**kwargs`` parameter must be a subtype of ``A``'s ``**kwargs``
+parameter::
+
+    class NoKwargs(Protocol):
+        def __call__(self) -> None: ...
+
+    class IntKwargs(Protocol):
+        def __call__(self, **kwargs: int) -> None: ...
+
+    class FloatKwargs(Protocol):
+        def __call__(self, **kwargs: float) -> None: ...
+
+    def func(no_kwargs: NoKwargs, int_kwargs: IntKwargs, float_kwargs: FloatKwargs):
+        f1: NoKwargs = int_kwargs  # OK
+        f2: NoKwargs = float_kwargs  # OK
+
+        f3: IntKwargs = no_kwargs  # Error: missing **kwargs parameter
+        f4: IntKwargs = float_kwargs  # OK
+
+        f5: FloatKwargs = no_kwargs  # Error: missing **kwargs parameter
+        f6: FloatKwargs = int_kwargs  # Error: float is not subtype of int
+
+If a callable ``B`` has a signature with one or more keyword-only parameters,
+a callable ``A`` is a subtype of ``B`` if ``A`` has a ``**kwargs`` parameter
+whose type is a supertype of the types of any otherwise-unmatched keyword-only
+parameters in ``B``::
+
+    class KwOnly(Protocol):
+        def __call__(self, *, a: int, b: str) -> None: ...
+
+    class IntKwargs(Protocol):
+        def __call__(self, **kwargs: int) -> None: ...
+
+    class IntStrKwargs(Protocol):
+        def __call__(self, **kwargs: int | str) -> None: ...
+
+    class StrKwargs(Protocol):
+        def __call__(self, *, a: int, **kwargs: str) -> None: ...
+
+    class Standard(Protocol):
+        def __call__(self, a: int, b: str) -> None: ...
+
+    def func(int_kwargs: IntKwargs, int_str_kwargs: IntStrKwargs, str_kwargs: StrKwargs):
+        f1: KwOnly = int_kwargs  # Error: str is not subtype of int
+        f2: KwOnly = int_str_kwargs  # OK
+        f3: KwOnly = str_kwargs  # OK
+        f4: IntStrKwargs = str_kwargs  # Error: int | str is not subtype of str
+        f5: IntStrKwargs = int_kwargs  # Error: int | str is not subtype of int
+        f6: StrKwargs = int_str_kwargs  # OK
+        f7: StrKwargs = int_kwargs  # Error: str is not subtype of int
+        f8: IntKwargs = int_str_kwargs  # OK
+        f9: IntKwargs = str_kwargs  # Error: int is not subtype of str
+        f10: Standard = int_str_kwargs  # Error: Does not accept positional arguments
+        f11: Standard = str_kwargs  # Error: Does not accept positional arguments
+
+Subtyping relationships for callable signatures that contain a ``**kwargs``
+with an unpacked ``TypedDict`` are described in the section :ref:`above <unpack-kwargs>`.
```

Comment:
Perhaps part of that section should be moved here, but that can wait for another day.